src/entities/Customer.java

```diff
@@ -1,13 +1,12 @@
 package entities;
 
-import org.json.JSONArray;
 import org.json.JSONObject;
 
 /**
  * The type Customer.
  */
 
-public class Customer extends User implements JSONable{
+public class Customer extends User {
```

Comment:
Why are you removing the "implements JSONable"?

Reply:
should user implement JSONable?

Reply:
I think user should implement JSONable so that vendor can get it too